person.rb

```diff
@@ -0,0 +1,32 @@
+require './nameable'
+class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Kindly add the method for creating a rental as demonstrated earlier, but this time with `book` and `date` objects as parameters. You will need to import Rental class here.